0001-typedb/01-start/02-installation.md

```diff
@@ -0,0 +1,328 @@
+---
+pageTitle: Installation
+keywords: setup, getting started, typedb, download, install, server, linux, mac, windows, docker
+longTailKeywords: typedb on linux, typedb on mac, typedb on windows, start typedb server
+summary: Install and run the TypeDB Server on Linux, Mac or Windows.
+toc: false
+---
+
+# Installation
+
+<!--- Reimplement tabs when they are product ready -->
+
+## Docker
+
+### Install
+
+1. Ensure [Docker](https://docs.docker.com/get-docker/) is installed and running:
+
+<!-- test-ignore -->
+   ```bash
+   docker version
+   ```
+
+2. Pull the latest TypeDB Docker image:
+
+<!-- test-ignore -->
+   ```bash
+   docker pull vaticle/typedb:latest
+   ```
+
+
+### Start
+
+The TypeDB container exposes TypeDB’s default port, `1729`, and uses its default data directory, 
+`/opt/typedb-all-linux/server/data`.
+
+The following command starts a named TypeDB container, maps its exposed port to the host machine and creates a named 
+volume mapped to the data directory — making it persistent across container restarts.
+
+<!-- test-ignore -->
+```bash
+docker run --name try-typedb -d -v try-typedb-data:/opt/typedb-all-linux/server/data/ -p 1729:1729 --platform linux/amd64 vaticle/typedb:latest
+```
+
+<div class="note">
+[Note]
+The `--platform linux/amd64` parameter is required to run the TypeDB container on MacOS with an ARM64 architecture 
+(e.g. M1 processor).
+</div>
+
+The following variables should be noted and can be modified:
+
+- `try-typedb` – the name of the container,
+- `try-typedb-data` – the name of the volume to persist data,
+- `latest` – the version of TypeDB.
+
+### Stop
+
+<!-- test-ignore -->
+```bash
+docker stop try-typedb
+```
+
+## Windows
+
+### Install
+
+1. Ensure Java 11+ is installed
+
+      TypeDB supports the [OpenJDK](https://jdk.java.net) and 
+      [Oracle JDK](https://www.oracle.com/java/technologies/downloads/).
+
+2. Download TypeDB
+   
+   Download the latest release of [TypeDB](https://vaticle.com/download).
+
+3. Extract archive
+   
+   Create a new directory, extract the contents of the zip file and move them (hereinafter replace `2.15.0` with the 
+   version downloaded):
+
+   <!-- test-ignore -->
+   ```shell
+   mkdir "C:\Program Files\TypeDB"
+   tar xvf ./Downloads/typedb-all-windows-2.15.0.zip
+   move ./Downloads/typedb-all-windows-2.15.0/* "C:\Program Files\TypeDB"
+   ```
+
+4. Update the PATH environment variable with the TypeDB installation directory:
+
+   <!-- test-ignore -->
+   ```shell
+   setx /M PATH "%path%;C:\Program Files\TypeDB"
+   ```
+
+### Start
+
+Run the following command in a terminal:
+
+<!-- test-ignore -->
+```shell
+typedb server
+```
+
+<div class="note">
+[Note]
+TypeDB will run in the foreground. If the terminal running TypeDB is closed, TypeDB will be shut down. We recommend 
```

Comment:
We could provide guidance on running the server in the background via `nohup typedb server&`, tmux or something else.